src/test/kotlin/lotto/domain/service/AutomaticLottoMachineTest.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain.service
+
+import lotto.domain.model.Lotto
+import lotto.domain.value.LottoCount
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+class AutomaticLottoMachineTest {
+    private val lottoMachine = AutomaticLottoMachine()
+
+    @Test
+    fun `로또 수량 5개에 대해 5장의 로또를 발급한다`() {
+        // given
+        val lottoCount = LottoCount(5)
+
+        // when
+        val lottos: List<Lotto> = lottoMachine.generate(lottoCount)
+
+        // then
+        Assertions.assertThat(lottos.size).isEqualTo(5)
```

Comment:
nit: 아래처럼 작성이 가능해요 🤠
```suggestion
        import org.assertj.core.api.Assertions.assertThat
        assertThat(lottos.size).isEqualTo(5)
```